deploy/examples/import-external-cluster.sh

```diff
@@ -197,11 +197,11 @@ parameters:
   dataPool: $RBD_METADATA_EC_POOL_NAME
   imageFormat: "2"
   imageFeatures: layering
-  csi.storage.k8s.io/provisioner-secret-name: $CSI_RBD_PROVISIONER_SECRET_NAME
+  csi.storage.k8s.io/provisioner-secret-name: rook-$CSI_RBD_PROVISIONER_SECRET_NAME
```

Comment:
Thanks for looking into this,
This is something very crucial thing that didn't get changed,

For testing locally, Probably I did the changes but didn't push it, 
@travisn can we think of investigating more on the CI for external-cluster so these small mistakes won't repeat.

This is an upstream-specific file so won't impact downstream.

Reply:
@parth-gr regarding more CI for the script, the issue is in bash script right not in python script? I'm asking since we are 
 moving forward with the python script and the python script has the latest features and more ci tests.

But good to fix the bugs in the script.

Reply:
Yes, we are moving forward with the python script right,

This is a different import script which is needed for upstream deployment after running the python file,

So I am asking a CI that will overall create an external cluster(osds,csi-pods, all thing on consumer) and also then go and creates underlying PVs for the Storage class created.

Reply:
@parth-gr I am working on a Helm chart for external Ceph cluster integration, where i will be replacing the import-external-cluster.sh with Helm templates. I will create a PR once finished, so we can discuss, if we merge it into the existing Chart or have a separate chart for it.

Reply:
Adding it to Helm is a good idea,

But probably you know that we need to also run the python script first and make sure if it can go with Helm configuration too,
If it works fine and can be merged so it would be a good idea to keep it with what's already present

Reply:
The python script is inevitable to get the information needed from the Ceph cluster. Instead of defining though these variables in the import script, we can define them inside the helm values.yaml file.